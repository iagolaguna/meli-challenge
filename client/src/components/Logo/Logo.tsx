import LogoML from "assets/Logo_ML.png";
import LogoML2x from "assets/Logo_ML@2x.png";

export const Logo = () => {
  return (
    <img
      src={LogoML}
      alt="Logo Mercado livre"
      srcSet={`${LogoML} 1200w, ${LogoML2x} 1900w`}
    />
  );
};
