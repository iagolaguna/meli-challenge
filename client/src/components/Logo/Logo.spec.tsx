import { render } from "@testing-library/react";
import { Logo } from "components/Logo/Logo";

describe("Logo", () => {
  it("should match snpashot", () => {
    const { container } = render(<Logo />);
    expect(container).toMatchSnapshot();
  });
});
