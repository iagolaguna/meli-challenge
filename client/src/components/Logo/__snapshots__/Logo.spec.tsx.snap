// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should match snpashot 1`] = `
<div>
  <img
    alt="Logo Mercado livre"
    src="Logo_ML.png"
    srcset="Logo_ML.png 1200w, Logo_ML@2x.png 1900w"
  />
</div>
`;
